operator/tasks/allocator_azure/allocator_azure.go

```diff
@@ -12,22 +12,33 @@
 // See the License for the specific language governing permissions and
```

Comment:
Why not `pkg/ipam/allocator/azure/`?

Reply:
As above, happy to move if we agree on what is the right place for it to live.